plugins/zoom/unit_test/test_monitor_sign_in_out_activity.py

```diff
@@ -301,3 +307,106 @@ def test_api_output_changed_error_catch(self, mock_call: MagicMock, mock_datetim
     def test_dedupe_events(self, events: List[Event], latest_event_time: str, expected: List[Event]) -> None:
         output = self.task._dedupe_events(all_events=events, latest_event_timestamp=latest_event_time)
         self.assertEqual(expected, output)
+
+    @parameterized.expand(
+        [
+            [
+                {
+                    "lookback": CUSTOM_LOOKBACK,
+                    "cutoff": CUSTOM_CUTOFF_DATE,
+                },
+                {
+                    "start_date": "2023-01-23T22:00:00Z",
+                    "end_date": "2023-02-23T22:00:00Z",
+                    "page_size": 300,
+                    "next_page_token": None,
+                },
+                STUB_DATETIME_LAST_CUTOFF_HOURS,
+            ],
+            [
+                {
```

Comment:
this second test looks like the same as the first?

Reply:
for this the first one uses 

```
{
  "lookback": CUSTOM_LOOKBACK,
  "cutoff": CUSTOM_CUTOFF_DATE,
},
```

The second is using 
```
{
  "lookback": CUSTOM_LOOKBACK,
  "cutoff": CUSTOM_CUTOFF_HOURS,
},
```

to ensure that both the date and hours being passed in will give the same outcome

Reply:
My bad, can't read 🤦  resolving this comment.